### DISCUSSION TITLE
RestSharp 107

### DISCUSSION THREAD

**Discussion by youst03** [abuzuhri/Amazon-SP-API-CSharp#194] [Ideas]
Any roadmap on supporting RestSharp 107?  Since RestSharp 107 is a breaking change vs 106 if there are other dependencies that are moved to 107  this project breaks.

**Comment by jshuster** [comment]
Hey all - @youst03 is correct - i tried to update this myself with newer version of RestSharp, but was unsuccessful.
The newest version of RestSharp is a breaking change, since this package relies on 106.x, but other projects in my solution are built on 107.x
Any thoughts on this? Thanks for your consideration...
